vgen/src/main/java/io/gapi/vgen/go/GoDiscoveryContext.java

```diff
@@ -130,4 +131,14 @@ public boolean isPageStreaming(Method method) {
     }
     return hasPageToken && hasNextPageToken;
   }
+
+  private static final ImmutableTable<String, String, String> API_VERSION_RENAME =
```

Comment:
Add a comment saying that we need this because in some cases there is a mismatch between X and Y version numbers. Also mention that it's a table of API name and version (as found in X)  to the renamed version as found in Y.

Add a TODO(pongad) to find a more sustainable solution to this.